src/model/kaniBinaryRunner.ts

```diff
@@ -1,15 +1,21 @@
 // Copyright Kani Contributors
 // SPDX-License-Identifier: Apache-2.0 OR MIT
-import { exec } from 'child_process';
+import { exec, execFile } from 'child_process';
```

Comment:
As far as I can see `exec` is now unused, apart from line 11 and 13 for example. Is it possible to remove it?